src/modules/project/pages/projectView/views/projectMainBody/components/GoalModal.tsx

```diff
@@ -10,10 +10,14 @@ import {
   Text,
   VStack,
 } from '@chakra-ui/react'
+import EmojiPicker from 'emoji-picker-react'
+import { Categories } from 'emoji-picker-react'
```

Comment:
build failing due to unused import

Reply:
Yes it's done, we decided not to go with the fluentui emojis that will take more efforts to implement